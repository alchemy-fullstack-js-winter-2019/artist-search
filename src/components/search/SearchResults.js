import React from 'react';
import PropTypes from 'prop-types';
import SearchedArtist from './SearchedArtist';

function SearchResults({ results }) {
    const searchList = results.map(searched => {
        const send = { name: searched.name, id: searched.id };
        return <li key={searched.id}> <SearchedArtist 
            values={send}
        /></li>;
    });
    return (
        <ul>{searchList}</ul>
    ); 
}

SearchResults.propTypes = {
    results: PropTypes.array,
};

export default SearchResults;
