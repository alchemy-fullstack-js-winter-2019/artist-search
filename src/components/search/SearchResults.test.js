import React from 'react';
import renderer from 'react-test-renderer';
import SearchResults from './SearchResults';

jest.mock('../services/musicbrainApi.js');

describe('SearchResults Tests', () => {
    it('matches a snapshot', () => {
        const tree = renderer.create(
            <SearchResults results={[]} />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
