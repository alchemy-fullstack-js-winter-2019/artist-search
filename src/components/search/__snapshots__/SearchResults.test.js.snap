// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResults Tests matches a snapshot 1`] = `<ul />`;
